studio/components/interfaces/TableGridEditor/TableGridEditor.tsx

```diff
@@ -61,17 +70,81 @@ const TableGridEditor: FC<Props> = ({
 }) => {
   const { meta, ui, vault } = useStore()
   const router = useRouter()
+  const { query } = router
```

Comment:
Any reason for using `router.query` here instead of grabbing `id` from `useParams()`?

Reply:
aah nope! no reason. changed this.

Reply:
Fair callout on the cramped space. The api page is currently this cramped as well. 

I made this client libs section full width in the table editor panel though, which helps I think: 

<img width="877" alt="CleanShot 2023-02-23 at 12 16 51@2x" src="https://user-images.githubusercontent.com/105593/220958327-0a9d4542-6360-4929-b92a-b3a24089c4e1.png">